components/salsa-2022/src/function.rs

```diff
@@ -30,15 +30,17 @@ mod store;
 mod sync;
 
 /// Function ingredients are the "workhorse" of salsa.
-/// They are used for tracked functions, for the "value" fields of tracked structs, and for the fields of input structs.
-/// The function ingredient is fairly complex and so its code is spread across multiple modules, typically one per method.
-/// The main entry points are:
+/// They are used for tracked functions, for the "value" fields of tracked structs, and for the
+/// fields of input structs. The function ingredient is fairly complex and so its code is
+/// spread across multiple modules, typically one per method. The main entry points are:
```

Comment:
I don't have a strong opinion about this, but the newlines in this comment were intentionally spaced this way to permit easier diffs when text is edited in the future.

Reply:
I'm guessing some kind of formatting tool adjusted these?

Reply:
Yes, rustfmt did this on autosave. And fmt is reverted now.